github/filecoin-project.yml

```diff
@@ -4750,8 +4746,6 @@ repositories:
     allow_update_branch: false
     archived: false
     collaborators:
```

Comment:
```suggestion
```
Because collaborators is empty, you now need to remove it.  This is annoying - it has snagged me too.